srcpkgs/cni-plugins/files/10-default.conf

```diff
@@ -0,0 +1,15 @@
+{
+	"cniVersion": "0.3.1",
+	"name": "default",
+	"type": "bridge",
+	"bridge": "cni0",
+	"isGateway": true,
+	"ipMasq": true,
+	"ipam": {
+		"type": "host-local",
+		"subnet": "10.22.0.0/16",
+		"routes": [
+			{ "dst": "0.0.0.0/0" }
+		]
+	}
+}
```

Comment:
Can I ask why we track this?

Reply:
Common configuration setup. Not shipped in the cni repos, but could be useful for some.

Reply:
Unfortunately if CNI updates their format, we are obsolete and might not know it.

Reply:
I have removed the sample configuration.